### ISSUE TITLE
Prepo.py says it can't find cudart64_100.dll

### ISSUE THREAD

**Issue by markwitt1** [Kyubyong/dc_tts#59]
When I am running prepo.py, it says:
`W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'cudart64_100.dll'; dlerror: cudart64_100.dll not found`

**Comment by markwitt1** [comment]
Fixed it myself :) if anyone has the same problem you need to install the 2018 version of cuda
https://developer.nvidia.com/cuda-10.0-download-archive

**Comment by Traincraft101** [comment]
Yeah, tensorflow (as of now) isn't compatible with cuda 10.1 I made the same mistake myself at first.
